test/JDBC/upgrade/14_3/schedule

```diff
@@ -290,3 +290,4 @@ BABEL-CHECK-CONSTRAINT-before-14_6
 BABEL-3640
 sys_server_principals_dep
 sys_database_principals_dep
+TestVariableDataLength
```

Comment:
Let's not add this test here since the error messages are changing in 15 compared to 14.

Reply:
Alright